globeandmail/sophi-for-wordpress
# Prevent tracking a duplicate event

### Description of the Change

This PR adds a transient with default TTL of 10 seconds for every new tracking event. If the transient exists and contain the same data, the tracking event is considered as duplicated call and will be skipped.

Closes #269 

### Verification Process

1. Install and activate: Sophi for WordPress, Debug Bar for Sophi and Yoast SEO plugins
2. Create new post or perform an update of an existing one.
3. Expected to see the single tracking request in Debug Bar 

### Checklist:






### Changelog Entry

> Fixed - Prevent duplicate CMS publish/update event tracking.
